prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -713,4 +732,166 @@ class ActionHistory {
             }
         }
     }
+
+    /**
+     * Creates a string that will be used to populate the action_response column of an action
+     * after that action has been completed
+     * @param options Message options passed in
+     * @param warnings Store of warnings generated during the pipeline
+     * @param errors Store of errors generated during the pipeline
+     * @param verbose If true, all item routing details will be included in the response
+     * @param actionHistory The ongoing action history instance being used
+     * @param report The report this response body is for
```

Comment:
Add @return to the kdoc

Reply:
What happens when report is null?

Reply:
return added
When report is null it doesn't include the report-specific data (id, timestamp, topic, reportItemCount)